code/modules/hydroponics/seed_datums.dm

```diff
@@ -1170,7 +1170,7 @@
 	name = "lemon"
 	seed_name = "lemon"
 	display_name = "lemon trees"
-	chems = list("plantmatter" = list(1,20))
+	chems = list("plantmatter" = list(1,20), "vitamin" = list(1,25))
 	mutants = list("cashtree")
 	kitchen_tag = "lemon"
 	preset_icon = "lemon"
```

Comment:
It should be a reagent that can be randomly picked, already; it's not explicitly blocked; it has `can_grow_in_plants` set to `1`.